tests_python/resources/_pydevd_test_find_main_thread_id.py

```diff
@@ -24,7 +24,7 @@ def wait_for_condition(condition, msg=None, timeout=5, sleep=.05):
 def check_main_thread_id_simple():
     import attach_script
     import sys
-    assert 'threading' not in sys.modules
+    # assert 'threading' not in sys.modules
```

Comment:
This was no longer true in 3.13.

Reply:
You can't run this from the debugger. This is run from `tests_python.test_utilities.test_find_main_thread_id` (where this is not an issue).